From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 23 May 2019 16:04:51 -0400
Subject: Use the older "import-show" with "--dry-run" instead of "show-only"

"--import-options import-show" was introduced in 2.1.14.

"--import-options show-only" was introduced in 2.1.23.

This gives broader compatibility.

The version test for the "import-show" feature is itself probably
wrong, but this patch doesn't address it.

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
---
 package/key.jsm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/key.jsm b/package/key.jsm
index 565273f..32ded1f 100644
--- a/package/key.jsm
+++ b/package/key.jsm
@@ -219,7 +219,7 @@ function getGpgKeyData(armorKeyString) {
 
   EnigmailLog.DEBUG("key.jsm: getGpgKeyData()\n");
   let command = EnigmailGpg.agentPath;
-  let args = EnigmailGpg.getStandardArgs(false).concat(["--no-tty", "--batch", "--no-verbose", "--dry-run", "--with-fingerprint", "--with-colons", "--import-options", "show-only", "--import"]);
+  let args = EnigmailGpg.getStandardArgs(false).concat(["--no-tty", "--batch", "--no-verbose", "--dry-run", "--with-fingerprint", "--with-colons", "--import-options", "import-show", "--dry-run", "--import"]);
 
   let statusFlagsObj = {};
   let statusMsgObj = {};
@@ -288,4 +288,4 @@ function getGpgKeyData(armorKeyString) {
   }
 
   return keyList;
-}
\ No newline at end of file
+}
